ans_task.yml

```
@@ -0,0 +1,109 @@
+---
+- name: Update packages and install Apache2
+  hosts: localhost
+  become: yes
+
+  tasks:
+    - name: Update package list
+      apt:
+        update_cache: yes
+
+    - name: Install Apache2
+      apt:
+        name: apache2
+        state: present
+
```

@shastish , you can add it as a single task
```suggestion
    - name: Install Apache2
      apt:
        name: apache2
        state: present
        update_cache: yes
```